backend/app/exporters/serializers/ead.rb

```diff
@@ -497,20 +497,40 @@ def serialize_digital_object(digital_object, xml, fragments)
         xml.daodesc{ sanitize_mixed_content(content, xml, fragments, true) } if content
       }
     elsif file_versions.length == 1
-        atts['xlink:type'] = 'simple'
-        atts['xlink:href'] = file_versions.first['file_uri'] || digital_object['digital_object_id']
-        atts['xlink:actuate'] = file_versions.first['xlink_actuate_attribute'] || 'onRequest'
-        atts['xlink:show'] = file_versions.first['xlink_show_attribute'] || 'new'
-        atts['xlink:role'] = file_versions.first['use_statement'] if file_versions.first['use_statement']
-        xml.dao(atts) {
-          xml.daodesc{ sanitize_mixed_content(content, xml, fragments, true) } if content
-        }
+      publish_file_uri = file_versions.first['file_uri'] && 
+                         (file_versions.first['publish'] == true || @include_unpublished)
+
+      atts['xlink:type'] = 'simple'
+
+      if publish_file_uri
+        atts['xlink:href'] = file_versions.first['file_uri'] 
+      else
+        atts['xlink:href'] = digital_object['digital_object_id']
```

Comment:
I don't know the exporters well enough to know this, but can the href just be set to nil?  As long as it will still export, this would give users the desired outcome of there being absolutely nothing (and thus no hyperlink) in the href.